src/components/Table/TableHeaderCell/TableHeaderCell.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { FC } from "react";
+import cx from "classnames";
+import { SubIcon, VibeComponentProps } from "../../../types";
+import styles from "./TableHeaderCell.module.scss";
+import Icon from "../../Icon/Icon";
+import IconButton from "../../IconButton/IconButton";
+import Info from "../../Icon/Icons/components/Info";
+import { ButtonType } from "../../Button/ButtonConstants";
+import Text from "../../Text/Text";
+import { getAriaSort, getNextSortState, getSortIcon } from "../Table/tableHelpers";
+
+export interface ITableHeaderCellProps extends VibeComponentProps {
+  title: string;
+  icon?: SubIcon;
+  infoContent?: string;
+  sortState?: "asc" | "desc" | "none";
```

Comment:
isn't "order" more clear?

Reply:
or direction

Reply:
also, I am not huge fan of the strings as enums method but if that's what Shlomi requested let's keep it